pipelines.adoc

```diff
@@ -161,19 +161,19 @@ oc apply -f <pipeline.yaml>
 
 === Run the pipeline
 
-A sample `manual-pipeline-run-template.yaml` file is provided in the link:https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator/manual-pipeline-runs[`/pipelines/manual-pipeline-runs` directory]. Rename the template file to `pipeline-run.yaml`, for example, and update the file to replace `collection-name` with the name of your collection. After you update the file, run it as shown in the following example.
+A sample `manual-pipeline-run-template.yaml` file is provided in the link:https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator/manual-pipeline-runs[`/pipelines/manual-pipeline-runs` directory]. Rename the template file to `pipeline-run.yaml`, for example, and update the file to replace `application-stack-name` with the name of your application stack. After you update the file, run it as shown in the following example.
 
 ----
-oc apply -f <collection-name>-pipeline-run.yaml
+oc apply -f <application-stack-name>-pipeline-run.yaml
 ----
 
-== Run pipelines from the command line for your custom built collections
+== Run pipelines from the command line for your custom built application stacks
 
-The following steps explain how to run pipelines against custom built collection images instead of the provided Kabanero collections.
+The following steps explain how to run pipelines against custom built application stack images instead of the provided application stacks.
 
-=== Set up a container registry URL for the custom collection image
+=== Set up a container registry URL for the custom application stack image
 
-By default, pipelines pull the collection images for Docker hub. If you are publishing your collection images to any other repository, use the following process to configure the custom repository from which your pipelines pull the container images.
+By default, pipelines pull the application stack images for Docker hub. If you are publishing your application stack images to any other repository, use the following process to configure the custom repository from which your pipelines pull the container images.
 
 . After you clone the `kabanero-pipelines` repository, find the `collection-image-registry-map.yaml` configmap template file. Add your container registry URL to this file in place of the `default-collection-image-registry-url` statement.
```

Comment:
collection-image-registry-map.yaml should be stack-image-registry-map.yaml (https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/sample-helper-files/stack-image-registry-map.yaml)

Reply:
fixed, along with several other instances nearby